mobile/test/common/integration/sds_integration_test.cc

```diff
@@ -19,9 +19,25 @@ const envoy::service::secret::v3::SdsDummy _sds_dummy;
 class SdsIntegrationTest : public XdsIntegrationTest {
 public:
   SdsIntegrationTest() {
+    override_builder_config_ = true;
```

Comment:
moving some of the shared code from xds to here because the rtds test has switched to using the builder API